src/components/time_picker_slider/index.utils.ts

```
@@ -0,0 +1,32 @@
+  if (isNaN(numHours) || numHours < 0 || numHours >= 24) {
+    return '00';
+  }
+
+  return formatTimeUnit(numHours);
+};
+
+export const validateMinutes = (minutes: number | string) => {
+  const numMinutes =
+    typeof minutes === 'string' ? parseInt(minutes, 10) : minutes;
+
+  if (isNaN(numMinutes) || numMinutes < 0 || numMinutes >= 60) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)